.expeditor/scripts/release_habitat/build_mac_hab_binary.sh

```diff
@@ -65,12 +62,10 @@ ${hab_binary} pkg upload \
               --auth="${HAB_AUTH_TOKEN}" \
               "results/${pkg_artifact:?}"
 
-${hab_binary} pkg promote \
-              --auth="${HAB_AUTH_TOKEN}" \
-              "${pkg_ident}" "${channel}" "${BUILD_PKG_TARGET}"
-
 echo "--- :buildkite: Storing artifact ${pkg_ident}"
 buildkite-agent artifact upload "results/${pkg_artifact}"
```

Comment:
Can we get rid of this, since we're storing these artifacts in builder now?

Reply:
Ah, yes.

Reply:
Addressed in https://github.com/habitat-sh/habitat/pull/7043/commits/6de3f6f7b23b51795c970fe1b2e64f70440112bd